wandb/data_types.py

```diff
@@ -143,6 +144,13 @@ def _set_file(self, path, is_tmp=False, extension=None):
     def get_media_subdir(cls):
         raise NotImplementedError
 
+    @classmethod
+    def captions(cls, media_items):
+        if media_items[0]._caption != None:
```

Comment:
Should we handle the case of no media_items?

Reply:
`seq_to_json` won't be called on an empty array.